synapse/tests/test_lib_stormtypes.py

```diff
@@ -2674,6 +2674,7 @@ async def getCronJob(text):
                 async with getCronJob("cron.at --day +1,+7 {$lib.queue.get(foo).put(at2)}"):
 
                     unixtime += DAYSECS
+                    core.agenda._wake_event.set()
```

Comment:
why are these set?

Reply:
There was a race in the test between the agenda scheduling loop runs and changing the time. This makes sure the agenda scheduling loop gets a shot after the time is changed.